troi.py

```diff
@@ -87,7 +97,7 @@ def info(patch):
     print()
     print("  expected inputs:")
     for input in inputs:
-        print("     %s, type %s" % (input[1], input[0]))
+        print(f"     {input['name']}, type {input['type']}: {input['desc']}")
```

Comment:
I saw that you used python 3.8 in the docker image, so took a risk and used f-strings. Let me know what you think about standardising on these